tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -1050,35 +1038,22 @@ var apiListCmdV2 = &cobra.Command{
         var retApiList *whisk.ApiListResponseV2
         var retApi *whisk.ApiGetResponseV2
         var retApiArray *whisk.RetApiArrayV2
+        var apiPath string
+        var apiVerb string
 
         if whiskErr := checkArgs(args, 0, 3, "Api list",
             wski18n.T("Optional parameters are: API base path (or API name), API relative path and operation.")); whiskErr != nil {
             return whiskErr
         }
 
-        props, _ := readProps(Properties.PropsFile)
-        spaceguid := strings.Split(props["AUTH"], ":")[0]
-        var accesstoken string = "DUMMY_TOKEN"
-        if len(props["APIGW_ACCESS_TOKEN"]) > 0 {
-            accesstoken = props["APIGW_ACCESS_TOKEN"]
-        }
-
-        // Get API request body
-        apiGetReq := new(whisk.ApiGetRequest)
-        apiGetReq.Namespace = client.Config.Namespace
-        // Get API request options
-        apiGetReqOptions := new(whisk.ApiGetRequestOptions)
-        apiGetReqOptions.AccessToken = accesstoken
-        apiGetReqOptions.SpaceGuid = spaceguid
-
-        // List API request query parameters
-        apiListReqOptions := new(whisk.ApiListRequestOptions)
-        apiListReqOptions.Limit = flags.common.limit
-        apiListReqOptions.Skip = flags.common.skip
-        apiListReqOptions.AccessToken = accesstoken
-        apiListReqOptions.SpaceGuid = spaceguid
-
         if (len(args) == 0) {
+            // List API request query parameters
+            apiListReqOptions := new(whisk.ApiListRequestOptions)
+            apiListReqOptions.Limit = flags.common.limit
+            apiListReqOptions.Skip = flags.common.skip
+            apiListReqOptions.SpaceGuid, _ = getUserContextId()
```

Comment:
@dubeejw - When an error (only if problem reading the configuration file) occurs, a default value is returned for each of these value.  Since the properties file is read earlier in the command flow, it's unlikely, but possible, the additional properties file read will fail when the earlier one does not.  The dummy values are:
- "DUMMY TOKEN" for the API GW access token, which will work fine with the open API GW
- "" for the Space GUID.  Since this is pulled from the auth key, having an error here but not elsewhere is unlikely; but just in case the logic flow changes; i'll add a quick check for the failure